azure-pipelines/build-python-packages.yml

```diff
@@ -6,7 +6,7 @@ stages:
 - stage: Build_Python_MacOS
   dependsOn: []
   variables:
-    VmImage: 'macOS-10.14'
+    VmImage: 'macOS-10.15'
```

Comment:
One side effect of bumping the build image version is that, because the `MACOSX_DEPLOYMENT_TARGET` isn't set anywhere, the Pythons and therefore any wheels produced will only be compatible with macOS >= 10.15. Is there a reason `MACOSX_DEPLOYMENT_TARGET` isn't used? The python.org builds use minimum macOS version from 10.6 to 10.9 (which admittedly is overkill).